src/subscriptions/Message.h

```diff
@@ -0,0 +1,38 @@
+#ifndef CLIO_SUBSCRIPTION_MESSAGE_H
+#define CLIO_SUBSCRIPTION_MESSAGE_H
+
+// This class should only be constructed once, then it can
+// be read from in parallel by many websocket senders
+class Message
+{
+    std::string message_;
+
+public:
+    Message() = delete;
+    Message(std::string&& message) : message_(message)
```

Comment:
Shouldn't it be `message_(std::move(message))`? to call the move constructor.

Reply:
duh, yes it should